src/submap.h

```diff
@@ -44,6 +44,12 @@ struct spawn_point {
         mission_id( MIS ), friendly( F ), name( N ) {}
 };
 
+struct partial_con {
```

Comment:
I think it makes more sense to put this in `construction.h` so we don't have to include submap in a bunch of places it doesn't really belong